modules/reference/pages/rpk/rpk-security/rpk-security-acl.adoc

```diff
@@ -60,8 +60,8 @@ When you create a role, you must bind or associate ACLs to it before it can be u
 
 == Resources
 
-A resource is what an ACL allows or denies access to. There are four resources
-within Redpanda: topics, groups, the cluster itself, and transactional IDs.
+A resource is what an ACL allows or denies access to. There are six resources
```

Comment:
in the SR AuthZ doc, the new ACL resources are listed as registry and subject. See https://github.com/redpanda-data/docs/pull/1224/files